hisensetv/__main__.py

```diff
@@ -77,7 +78,8 @@ def main():
         help="Do not connect with SSL (required for some models).",
     )
     parser.add_argument(
-        "-v", "--verbose", action="count", default=0, help="Logging verbosity."
+        "-v", "--
```

Comment:
This newline is invalid syntax, the rest looks good though.

Reply:
sorry I commited from a command line and had no idea about this newline. should be fixed now